core/google/api/core/page_iterator.py

```diff
@@ -386,50 +364,44 @@ def _get_query_params(self):
     def _get_next_page_response(self):
         """Requests the next page from the path provided.
 
-        :rtype: dict
-        :returns: The parsed JSON response of the next page's contents.
+        Returns:
+            dict: The parsed JSON response of the next page's contents.
 
-        :raises ValueError: If the HTTP method is not ``GET`` or ``POST``.
+        Raises:
+            ValueError: If the HTTP method is not ``GET`` or ``POST``.
         """
         params = self._get_query_params()
         if self._HTTP_METHOD == 'GET':
-            return self.client._connection.api_request(
+            return self.api_request(
                 method=self._HTTP_METHOD,
                 path=self.path,
                 query_params=params)
         elif self._HTTP_METHOD == 'POST':
-            return self.client._connection.api_request(
+            return self.api_request(
                 method=self._HTTP_METHOD,
                 path=self.path,
                 data=params)
         else:
             raise ValueError('Unexpected HTTP method', self._HTTP_METHOD)
 
 
-class GAXIterator(Iterator):
+class _GAXIterator(Iterator):
```

Comment:
Per commit message:

> Make GAXIterator private- it will soon be removed.

The plan is to introduce a gRPCIterator that gapic will use, so that GAXIterator never comes into the picture.